fk7263/src/main/java/se/inera/intyg/common/fk7263/rest/Fk7263ModuleApi.java

```diff
@@ -713,4 +713,11 @@ public CertificateMessagesProvider getMessagesProvider() {
         return DefaultCertificateMessagesProvider.create(validationMessages);
     }
 
+    @Override
+    public String getUtlatandeToInternalModelResponse(Utlatande utlatande) throws ModuleException {
+        if (utlatande instanceof Fk7263Utlatande) {
+            return toInteralModelResponse((Fk7263Utlatande) utlatande);
```

Comment:
Spelling error... maybe can fix now, even if it seems to exist from before.... toInteralModelResponse -> toInternalModelResponse.

Reply:
Done.